### PR TITLE
feat: update cta block to new design

### PR DESCRIPTION
**Pull request by paulpopus**
Updating the CTA block to the new style

Previous:
image
image

New:
image
image

Notes, the crosses at mobile should be above buttons (despite figma showing otherwise)

### CODE REVIEW

File: src/blocks/CallToAction/index.tsx [hunk 1]
```diff
@@ -24,32 +26,46 @@ export const CallToAction: React.FC<CallToActionProps> = props => {
...
+                className={[classes.scanline, 'cols-16 start-4 start-l-3 cols-m-8 start-m-1']
+                  .filter(Boolean)
+                  .join(' ')}
+                crosshairs={['top-left', 'bottom-left']}
+              />
+              {/* Double printing the BackgroundGrid component here so that it displays above the scanline */}
+              <BackgroundGrid ignoreGutter />
+              <CrosshairIcon className={[classes.crosshairTopLeft].filter(Boolean).join(' ')} />
+              <CrosshairIcon className={[classes.crosshairBottomRight].filter(Boolean).join(' ')} />
               {feature === 'cpa' && (
```
Comment (PatrikKozak) [inline src/blocks/CallToAction/index.tsx:50]: @paulpopus Is this `feature` prop / `CreatePayloadApp` component staying in the CTA? It looks pretty janky when it's rendered right now.

Screen Shot 2024-01-30 at 10 42 08 AM

File: src/blocks/CallToAction/index.tsx [hunk 2]
```diff
@@ -24,32 +26,46 @@ export const CallToAction: React.FC<CallToActionProps> = props => {
...
               <RichText content={richText} className={classes.content} />
             </div>
-            <div className={'cols-7 start-11 start-l-10 cols-m-8 start-m-1'}>
+            <div
+              className={[classes.linksContainer, 'cols-8 start-9 cols-m-8 start-m-1 grid']
+                .filter(Boolean)
+                .join(' ')}
+            >
+              <BackgroundScanline
+                className={[classes.scanline, 'cols-16 start-4 start-l-3 cols-m-8 start-m-1']
```
Comment (PatrikKozak) [inline src/blocks/CallToAction/index.tsx:41]: I think I would do `cols-16 start-5 cols-m-8 start-m-1` instead for the `BackgroundScanline`.

Designs shows it starting at exactly the half way point between the two grid lines.

File: src/blocks/CallToAction/index.module.scss [hunk 3]
```diff
@@ -1,43 +1,54 @@
...
     }
+  }
 }
 
-.contentWrap {
-    position: relative;
-    padding: calc(var(--base) * 4) 0;
+.scanline {
+  top: -20%;
+  height: 140%;
```
Comment (PatrikKozak) [inline src/blocks/CallToAction/index.module.scss:31]: Try `top: -25%` & `height: 150%;`

This to me looks more inline with the design

**Comment by PatrikKozak** [comment]
@paulpopus Is the `content` in mobile meant to have the `left` & `right` `padding` or should the content be up against the grid lines?

Screen Shot 2024-01-30 at 10 57 16 AM

File: src/blocks/CallToAction/index.module.scss [hunk 4]
```diff
@@ -1,43 +1,54 @@
...
-    width: 100%;
-    height: 100%;
-    position: absolute;
-    top: 0;
-    left: 0;
-}
+.container {
+  position: relative;
+  padding-top: calc(var(--base) * 10);
+  padding-bottom: calc(var(--base) * 10);
```
Comment (PatrikKozak) [inline src/blocks/CallToAction/index.module.scss:10]: @paulpopus `padding` here is too big in mobile - let's decrease it for `mid` / `small` breakpoints

Screen Shot 2024-01-30 at 10 58 51 AM

File: src/components/Button/index.module.scss [hunk 5]
```diff
@@ -17,11 +17,23 @@ $curve: cubic-bezier(.165, .84, .44, 1);
```
Comment (PatrikKozak) [inline src/components/Button/index.module.scss:17]: @paulpopus Button changes look fine - did you verify that nothing needed to be changed with the previous blocks that have been built? e.g. `card-grid`, your `sticky-list` block, `FeaturedMediaGallery` block?

Not extremely crucial but if its not taken care of now, then its just another thing we need to add to the list once review process starts

File: src/blocks/CallToAction/index.tsx (hunk 1 above)
Comment (paulpopus) [reply src/blocks/CallToAction/index.tsx:50]: Removed

**Pull request merged**